.gitignore

```diff
@@ -6,6 +6,8 @@ venv
 attachments/
 indigo_app/static/lime/develop
 staticfiles/
+static/stylesheets/app.css
+static/stylesheets/app.css.map
```

Comment:
I think these both need to be `indigo_app/static/...` ?

Reply:
Indeed.